### ISSUE TITLE
One little question to ask is it safe?

### ISSUE THREAD

**Issue by RajDevansh** [moom825/Discord-RAT-2.0#114]
i am in high school and not very educated on these things but is it safe?
also when downloading it is showing harmful stopped.why? any reasons?

**Comment by RajDevansh** [comment]
Also I am not quite getting it.can u make some sort of video on it?

**Comment by trollingboygamingYT** [comment]
i would recommend https://www.youtube.com/watch?v=-nsALpaFZ8U

**Comment by RajDevansh** [comment]
> i would recommend https://www.youtube.com/watch?v=-nsALpaFZ8U

will it tell me how to make it?

**Comment by trollingboygamingYT** [comment]
uh yeah it will but just use a VM to because i installed it on my main pc got a virus warning and i know it is a false positive but just use a VM and install this on it and then just look further in the video and you will learn how to make it.

**Comment by RajDevansh** [comment]
ok thanks bro for helping

**Comment by RajDevansh** [comment]
> uh yeah it will but just use a VM to because i installed it on my main pc got a virus warning and i know it is a false positive but just use a VM and install this on it and then just look further in the video and you will learn how to make it.

by any chance can i get your insta to contact for further info?

**Comment by trollingboygamingYT** [comment]
uh sure henninnoah but i havent installed this thing for myself havent tried it yet

**Comment by RajDevansh** [comment]
ohh.ok
i won't have any reverse effects of it affecting my own pc? right?

**Comment by trollingboygamingYT** [comment]
if you download this on a vm and even if you download it on youre main pc i dont think anything will happen but for security reasons i would download it on a vm

**Comment by RajDevansh** [comment]
okk thanks

**Comment by RajDevansh** [comment]
i tried through the video but it is not working.is it because i am directly launching client built intead of its image? or is it something else?

**Comment by ShadowViper466** [comment]
i think you just have to launch the image on a vm and then on the discord on youre main pc should open a session saying that a desktop connected

**Comment by trollingboygamingYT** [comment]
sorry ShadowViper466 is my other account

**Comment by KitKit20000** [comment]
guys the thing is down

yall should try quasar if you wanna troll your friends

**Comment by ShadowViper466** [comment]
okay thanks

**Comment by NFS11** [comment]
@KitKit20000 quasar doesn't have many features to troll so use xeno instead lol.

**Comment by Magmabros** [comment]
Sorry, I was trying to prepare a cybersecurity demo following a tutorial but when I ran the "build" instead of activating the bot this happened to me (sorry, English is not my native language)

![Image](https://github.com/user-attachments/assets/df51a140-697a-49e8-a4aa-a04bab56c661)

**Comment by NFS11** [comment]
yeah it's safe and you can check source code. do not share on discord because it won't allow it.
